bokehjs/src/coffee/models/plots/plot.coffee

```diff
@@ -239,7 +314,8 @@ class Plot extends LayoutDOM.Model
       # ALL BELOW ARE FOR PLOT CANVAS
       plot_width:        [ p.Number,   600                    ]
       plot_height:       [ p.Number,   600                    ]
-      title:             [ p.Instance, () -> new Title.Model({text: ""})]
+
+      title:             [ p.Any, () -> new Title.Model({text: ""})] # TODO: p.Either(p.Instance(Title), p.String)
```

Comment:
I expect it's just timing, still needs to be updated

> On Jun 17, 2016, at 19:35, Sarah Bird <notifications@github.com> wrote:
> 
> In bokehjs/src/coffee/models/plots/plot.coffee:
> 
> > @@ -239,7 +314,8 @@ class Plot extends LayoutDOM.Model
> >        # ALL BELOW ARE FOR PLOT CANVAS
> >        plot_width:        [ p.Number,   600                    ]
> >        plot_height:       [ p.Number,   600                    ]
> > -      title:             [ p.Instance, () -> new Title.Model({text: ""})]
> > +
> > +      title:             [ p.Any, () -> new Title.Model({text: ""})] # TODO: p.Either(p.Instance(Title), p.String)
> This is inconsistent with the python API, can you share some context.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>